packages/shared/src/lib/featureValues.ts

```diff
@@ -25,3 +25,11 @@ export enum PostPageOnboarding {
   Control = 'control',
   V1 = 'v1',
 }
+
+export enum OnboardingV4dot5 {
+  Control = 'control',
+  V4dot5 = 'v4dot5',
+}
+
+export const isOnboardingV4dot5 = (flag: string): boolean =>
```

Comment:
Do we want to add this here or in any utils? Seems like it messes one of the only files we have structured 😅

Reply:
I know it's not the best place. This was one of the places we didn't have a common file 😬 Maybe move it into the neighboring `func.ts` file?

Reply:
Do you really need this though, you can just use `===`? I don't think this helper adds anything.

Reply:
Just DX I guess, quicker to auto complete on isOnboardingV4dot5 than writing the === check. I'm indifferent really.

Reply:
I'd say we either move or do what Ante says it's what we commonly do ATM.

Reply:
Hmm for me DX would be to then have the `useFeature` and other logic wrapped so you don't need to import `features`, `useFeature`, your helper and then use it. Eg. how we have `useFeatureFeedLayoutV1` or `useStreakExperiment`.

At this point this just adds another function call to the stack. So I would personally prefer not to add anything but if you want the hook would be better option I guess.

Reply:
I'll remove it.